docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -78,81 +192,125 @@ elastic-agent.exe run
+When started successfully, {metricbeat} logs are stored in
+`data/logs/metricbeat` under the folder where {agent} was started. If that log
+path does not exist, the agent was unable to start {metricbeat}, which is a
+higher level problem to triage.
 
 [float]
 [[what-is-my-agent-config]]
-== I need to know the configuration used in the running {agent}
+== What configuration is the {agent} running?
 
-You can inspect the files in the folder from where you ran {agent}. The `elastic-agent.yml` file will have an uncommented citation if it is in '{fleet}' mode:
+To find the configuration file, inspect the `elastic-agent.yml` file in the
+folder where you ran {agent}. If you're running the agent in {fleet} mode, this
+file contains the following citation:
+
+[source,yaml]
+----
 Management: mode: "fleet"
+----
+
+The `action_store.yml` contains the entire, unencrypted configuration:
 
-The `action_store.yml` has the whole configuration, unencrypted. Look for 'outputs:hosts' section with elasticsearch location. This file will also have the current versions of the packages in use by the current configuration. The version of the {agent} is in the download folder / zip.
+* To see the {es} location, look at `outputs:hosts`.
+* To see the {agent} version, look at the download folder and zip filenames.
+
+This file also shows the version of all packages used by the current
+configuration.
 
 [float]
 [[where-is-the-data-agent-is-sending]]
-== I'm not sure {ingest-manager} app is showing the data my {agent} is sending
+== Why can't I see the data {agent} is sending?
 
-I think I have {agent} set up and running and there are no errors on the agent logs and I still don't see data in {kib}. The recommended way to assess this is to go straight to {es} data queries, or maybe just use {kib} Discover app. Look for any data in the index your {agent} is sending to (metrics-* for example) and see if the name.host of your host machine shows. If not, then perhaps the data is getting trapped / blocked in the network or isn't sent off of the {agent} host correctly in the first place due to some Firewall or security reason. While it should be redundant, it is all the same potentially helpful to install the stand-alone Metricbeat (see the Elastic docs for information) to see if that works to send data into ES, which would indicate a bug or problem in the {agent} side.
+If {elastic-agent} is set up and running, but you don't see data in {kib}:
 
-[float]
-[[i-deleted-my-agent]]
-== I deleted {agent} in the {fleet} {ingest-manager} app how can I get it back?
+//REVIEWERS: Is there a better command that we should recommend to users here?
+//I think it's good to show an actual command because we could have a newbie
+//reading this topic.
 
-Its ok, we got your back!  The data is still in {es}. And get the {agent} back in the UI, you will need only to stop, then re-enroll the {agent} on the host and run it again.
```

Comment:
I was trying so hard to be Elastic and put some humor in, I'm ok if you want to leave it out tho. Minimal sad panda tho.  About the command, its the same command ./elastic-agent run that is listed above we can re-cite it (or put it to a variable if its going to be repeated more) but that detail is up to you.

Reply:
Oh I know. I kept the comment...it's just a bit lower down. We just want to be a little careful about humor in troubleshooting topics. I think it works, tho, so let's go with it.

Reply:
About the comment...I was referring to the API call, not a command. My bad. I'll leave it as-is since you didn't flag it. Thanks!